newrelic.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config.defaults.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  app_name: ['cp-organisations-service'],
+  agent_enabled: false, // set via NEW_RELIC_ENABLED for production
+  license_key: '', // set via NEW_RELIC_LICENSE_KEY
```

Comment:
Is the key already set on K8s, so that i can merge brainlessly?

Reply:
yes